derp/derp_test.go

```diff
@@ -785,6 +790,72 @@ func TestForwarderRegistration(t *testing.T) {
 	})
 }
 
+type channelFwd struct {
+	id int
```

Comment:
what's this ID being used for? just debug prints, or?

Reply:
It's to make sure that different instances that reference the same channel hash to different values. I've added a comment to clarify.